src/common/utils/os-detect.h

```diff
@@ -2,7 +2,7 @@
 
 #include <winrt/Windows.Foundation.Metadata.h>
 
-// The following three helper functions determine if the user has a build version higher than or equal to 19h1 (aka 1903), as that is a requirement for xaml islands
+// The following three helper functions determine if the user has a build version greater than or equal to 19h1 (aka 1903), as that is a requirement for xaml islands
```

Comment:
I'm still experimenting with this rule, but in general `greater than or equal to` is special phrase, so while I wouldn't change `higher than` in some cases where there isn't a strong argument, I am proposing it here.